plugins/techdocs/src/home/components/TechDocsCustomHome.tsx

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import { useAsync } from 'react-use';
+import { makeStyles } from '@material-ui/core';
+
+import { catalogApiRef, CatalogApi } from '@backstage/plugin-catalog-react';
+import { Entity } from '@backstage/catalog-model';
+import {
+  CodeSnippet,
+  Content,
+  ConfigApi,
+  configApiRef,
+  Header,
+  HeaderTabs,
+  Page,
+  Progress,
+  useApi,
+  WarningPanel,
+  SupportButton,
+  ContentHeader,
+} from '@backstage/core';
+import { DocsTable } from './DocsTable';
+import { DocsCardGrid } from './DocsCardGrid';
+
+const widgets = {
+  DocsTable: DocsTable,
+  DocsCardGrid: DocsCardGrid,
+};
+
+export type WidgetType = 'DocsCardGrid' | 'DocsTable';
```

Comment:
@adrianke77 I had a chance to huddle with the rest of the core TechDocs team about the naming of this here and throughout.

In our roadmap, we have something planned called the [TechDocs Widget Framework](https://backstage.io/docs/features/techdocs/techdocs-overview#project-roadmap), which is specifically aimed at improving extensibility of the TechDocs experience itself (less so the TechDocs homepage).

We're not 100% sure what that'll end up being called, but we'd like to reserve the "widget" name for that future functionality.

Would you mind using a different name?  Perhaps `panel` or `pane` or something similar?

Reply:
Ok will rename 'widgets' to 'panels'

Reply:
Updated for above